charts/nuts-node/templates/nuts-data-pv.yaml

```diff
@@ -8,22 +9,26 @@ metadata:
     "helm.sh/resource-policy": keep
 spec:
   persistentVolumeReclaimPolicy: Retain
-  storageClassName: ""
+  storageClassName: {{ .Values.nuts.data.persistedVolume.storageClassName | default "" }}
   capacity:
     storage: {{ .Values.nuts.data.persistedVolume.capacity | default "100Mi" }}
   accessModes: {{ required "Please define `nuts.data.persistedVolume.accessModes` in `values.yaml`" .Values.nuts.data.persistedVolume.accessModes }}
   hostPath:
-    path: {{ .Values.nuts.data.persistedVolume.mountPath | default "/opt/nuts/data" }}
+    path: {{ .Values.nuts.data.persistedVolume.mountPath  }}
 ---
+{{ end }}
+{{ if .Values.nuts.data.persistedVolumeClaim.enabled }}
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: nuts-data-pvc
   annotations:
     "helm.sh/resource-policy": keep
 spec:
-  storageClassName: "" # Empty string must be explicitly set otherwise default StorageClass will be set
-  accessModes: {{ required "Please define `nuts.data.persistedVolume.accessModes` in `values.yaml`" .Values.nuts.data.persistedVolume.accessModes }}
+  storageClassName: {{ .Values.nuts.data.persistedVolumeClaim.storageClassName | default "" }}
```

Comment:
If GKE needs a storage class name, can we provide a default or warn the user about a missing name?

Reply:
For GKE empty ("") means the default storage class. Currently the default storage class causes problems, because the default is read only. I consider this to be specific to GKE and something the deployer will run into at an early stage. Also, GKE might change this behaviour in the future, or is there a possibility to change the default storage class that we have missed somehow.. 

In general, this PR is about the ability to change the storage class. Without this ability, it is impossible to deploy to GKE. A warning should only be in place if all platforms would suffer from this.